<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChemicalPassportForm</class>
 <widget class="QWidget" name="ChemicalPassportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0,0,1">
   <property name="spacing">
    <number>14</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="detectorLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="detectorLabel">
          <property name="text">
           <string>Detector</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="detectorCombo"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="sampleInjVolLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="sampleInjVolLabel">
          <property name="text">
           <string>Sample injection volume, µl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="sampleInjVolSpin">
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="columnToggleBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: left;}</string>
     </property>
     <property name="text">
      <string> Column</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="columnWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="innerDiameterLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="innerDiameterLabel">
          <property name="text">
           <string>Inner diameter, mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="innerDiameterSpin">
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="staticPhaseThicknessLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="staticPhaseThicknessLabel">
          <property name="text">
           <string>Static phase thickness, µl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="staticPhaseThicknessSpin">
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="staticPhaseCompositionLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="staticPhaseCompositionLabel">
          <property name="text">
           <string>Static phase composition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="staticPhaseCompositionLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="columnLengthLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="columnLengthLabel">
          <property name="text">
           <string>Column length, m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="columnLengthSpin">
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="columnMarkLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="columnMarkLabel">
          <property name="text">
           <string>Column mark</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="columnMarkLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="tempToggleBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: left;}</string>
     </property>
     <property name="text">
      <string> Temperature regime</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="temperatureWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="columnTepmRiseRateLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="columnTepmRiseRateLabel">
          <property name="text">
           <string>Column t rise rate, C*min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="columnTepmRiseRateSpin">
          <property name="minimum">
           <double>-300.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="initialColumnTempLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="initialColumnTempLabel">
          <property name="text">
           <string>Initial column t, C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="initialColumnTempSpin">
          <property name="minimum">
           <double>-300.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="detectorTempLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="detectorTempLabel">
          <property name="text">
           <string>t detector C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="detectorTempSpin">
          <property name="minimum">
           <double>-300.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="evaporatorTempLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="evaporatorTempLabel">
          <property name="text">
           <string>t evaporator, C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="evaporatorTempSpin">
          <property name="minimum">
           <double>-300.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="columnHoldingTimeAtMaxTempLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="columnHoldingTimeAtMaxTempLabel">
          <property name="text">
           <string>Column holding time at maximum t, min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="columnHoldingTimeAtMaxTempSpin">
          <property name="minimum">
           <double>-300.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="additionalToggleBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: left;}</string>
     </property>
     <property name="text">
      <string>Additional parameters</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="additionalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="gasCarrierFlowLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="gasCarrierFlowLabel">
          <property name="text">
           <string>Gas-carrier flow, ml/min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="gasCarrierFlowSpin">
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="analysisDateLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="analysisDateLabel">
          <property name="text">
           <string>Analysis date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="analysisDateEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="samplePreparationTypeLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="samplePreparationTypeLabel">
          <property name="text">
           <string>Sample preparation type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="samplePreparationTypeCombo"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="flowSeparationLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="flowSeparationLabel">
          <property name="text">
           <string>Flow separation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="flowSeparationLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="gasCarrierTypeLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="gasCarrierTypeLabel">
          <property name="text">
           <string>Gas-carrier type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="gasCarrierTypeCombo"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="innerStandardLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="innerStandardLabel">
          <property name="text">
           <string>Inner standard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="innerStandardLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QVBoxLayout" name="additionalInfoLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="additionalInfoLabel">
          <property name="text">
           <string>Additional information</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="additionalInfoLineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>446</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>detectorCombo</tabstop>
  <tabstop>sampleInjVolSpin</tabstop>
  <tabstop>columnToggleBtn</tabstop>
  <tabstop>columnLengthSpin</tabstop>
  <tabstop>innerDiameterSpin</tabstop>
  <tabstop>staticPhaseThicknessSpin</tabstop>
  <tabstop>staticPhaseCompositionLineEdit</tabstop>
  <tabstop>columnMarkLineEdit</tabstop>
  <tabstop>tempToggleBtn</tabstop>
  <tabstop>evaporatorTempSpin</tabstop>
  <tabstop>detectorTempSpin</tabstop>
  <tabstop>initialColumnTempSpin</tabstop>
  <tabstop>columnTepmRiseRateSpin</tabstop>
  <tabstop>columnHoldingTimeAtMaxTempSpin</tabstop>
  <tabstop>additionalToggleBtn</tabstop>
  <tabstop>gasCarrierFlowSpin</tabstop>
  <tabstop>flowSeparationLineEdit</tabstop>
  <tabstop>innerStandardLineEdit</tabstop>
  <tabstop>gasCarrierTypeCombo</tabstop>
  <tabstop>samplePreparationTypeCombo</tabstop>
  <tabstop>additionalInfoLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
